<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessVarGui</class>
 <widget class="QDialog" name="ProcessVarGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Process Variables</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="TextLabel4">
       <property name="text">
        <string>CAN Identifier</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="canopenPVarViewIdentifier"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PushButton1">
       <property name="text">
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel5">
     <property name="text">
      <string>Rx (Receive) [dec]</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3Table" name="rxTable" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="numRows" stdset="0">
      <number>4</number>
     </property>
     <property name="numCols" stdset="0">
      <number>10</number>
     </property>
     <property name="showGrid" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel6">
     <property name="text">
      <string>Tx (Transmit) [dec]</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3Table" name="txTable" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="numRows" stdset="0">
      <number>4</number>
     </property>
     <property name="numCols" stdset="0">
      <number>10</number>
     </property>
     <property name="showGrid" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>QWidget</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>canopenPVarViewIdentifier</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProcessVarGui</receiver>
   <slot>canopenIdChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PushButton1</sender>
   <signal>clicked()</signal>
   <receiver>ProcessVarGui</receiver>
   <slot>canopenIdChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
